### PR TITLE
LushEditText is added based on latest form field format Joss done

### PR DESCRIPTION
**Pull request by gokmev**

### CODE REVIEW

File: views/src/main/java/com/lush/view/LushEditText.java [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+
+	@Override
+	public void onRestoreInstanceState(Parcelable state)
+	{
+		if (state instanceof Bundle)
+		{
+			Bundle bundle = (Bundle) state;
+			isErrorVisible = bundle.getBoolean(KEY_ERROR_VISIBLE);
+			state = bundle.getParcelable("superState");
+			isManualTextChange = false;
```
Comment (JamieCruwys) [inline views/src/main/java/com/lush/view/LushEditText.java]: Nice!

File: views/src/main/java/com/lush/view/LushEditText.java [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+			titleView.setVisibility(VISIBLE);
+		}
+	}
+
+	public Editable getText()
+	{
+		return editText.getText();
+	}
+
+
```
Comment (JamieCruwys) [inline views/src/main/java/com/lush/view/LushEditText.java]: Remove double new line

File: views/src/main/java/com/lush/view/LushEditText.java [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+		redExclamationImage.setLayoutParams(exclamationLayoutParams);
+		editText.addTextChangedListener(this);
+		editText.setHint(hint);
+		isManualTextChange = false; // setInputType triggers automated text change
+		editText.setInputType(inputType);
+		editText.setTextAppearance(context, R.style.Text_Black_HelveticaRoman);
+		RelativeLayout.LayoutParams editTextLayoutParams = (RelativeLayout.LayoutParams) editText.getLayoutParams();
+		editTextLayoutParams.height = (int) height;
+		editText.setLayoutParams(editTextLayoutParams);
+		if (TextUtils.isEmpty(title))
```
Comment (gokmev) [inline views/src/main/java/com/lush/view/LushEditText.java]: Done

File: views/src/main/java/com/lush/view/LushEditText.java [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+
+	public void setTitle(CharSequence title)
+	{
+		if (TextUtils.isEmpty(title))
+		{
+			titleView.setVisibility(GONE);
+		}
+		else
+		{
+			titleView.setVisibility(VISIBLE);
```
Comment (gokmev) [inline views/src/main/java/com/lush/view/LushEditText.java]: Done

File: views/src/main/java/com/lush/view/LushEditText.java (hunk 2 above)
Comment (gokmev) [inline views/src/main/java/com/lush/view/LushEditText.java]: Done

**Pull request closed without merging**